### ISSUE TITLE
Email Features

### ISSUE THREAD

**Issue by vabene1111** [vabene1111/recipes#626]
- [x] Signup with email confirmation
- [x] Password reset
- [x] Cooldowns on email sending https://github.com/pennersr/django-allauth/pull/2631
- [x] Ability to properly disable password reset and signup (also on the frontend)
- [x] example.com in messages
- [x] ability to re-send and check email verification status

**Comment by vabene1111** [comment]
TODO: 

- [ ] Cooldowns on email sending https://github.com/pennersr/django-allauth/pull/2631
- [ ] Ability to properly disable password reset and signup (also on the frontend)

**Comment by vabene1111** [comment]
example.com related to site name (can be changed in django admin)

**Comment by vabene1111** [comment]
since allauth's does not seem to merge the proper rate limiting PR the following mitigation could work
- captchas on signup/reset
- override allauth adapter and cache last send time for any email address, add cooldown here

**Comment by 9mido** [comment]
Suggestion would be to use hcaptcha you can get revenue and it is better for privacy.

If you figure out the adapter and cache please share it with the rest of the allauth community to see how you did it.

**Comment by vabene1111** [comment]
sure, i will create a little explaination when i find a solution

thanks for the captcha hint, i will have to look into what is currently privacy friendly and working well but that seems like good solution

**Comment by vabene1111** [comment]
@9mido i have posted a solution to the problem over in the original post. Its not great but it works for now. You can also see it in this repo.

**Comment by sam9032** [comment]
Hey there, sorry for posting on this closed issue. First things first, thank you for this amazing piece of software! I got really excited as I found your software last week and installed it.
The confirmation email contains the example.com entry and that's why I stumbled upon this issue here. I have now adjusted the "Website" entry in the Django Admin Menu and hope that this is how it works. Is there a way to set the domain in the .env file?
It would also be helpful to place a hint in the documentary, maybe I just overlooked it?
